.github/workflows/ci.yml

```diff
@@ -279,3 +281,25 @@ jobs:
 
             If this is not what was expected, commit changes to `config/versions.json`.
             </details>
+
+      - name: PR Description Notifier
+        env:
+          GH_TOKEN: ${{ github.token }}
+          PR_BODY_FILE_LOCATION: ./body.txt
+          LATEST_PRERELEASE_SECTION_REGEX: "^:rocket: .* :rocket:$"
+          # We can't use many punctuation marks in the Prerelease Section because `sed` treats
+          # the replacement string as a regex string - no special regex chars without escaping
+          LATEST_PRERELEASE_SECTION: ":rocket: The latest prerelease `${{ needs.defaults.outputs.root-sbd }}/${{ needs.check-spack-yaml.outputs.prerelease }}` is here: ${{ steps.comment.outputs.comment-link }} :rocket:"
+        run: |
+          gh pr view ${{ github.event.pull_request.number }} --repo ${{ github.repository }} --json body --jq .body > ${{ env.PR_BODY_FILE_LOCATION }}
+
+          if grep -q '${{ env.LATEST_PRERELEASE_SECTION_REGEX }}' ${{ env.PR_BODY_FILE_LOCATION }}; then  # there is an existing prerelease section
+            # Replace the existing prerelease section - we use '|' as a delimiter because there are '/' in the replacement string
+            sed -i 's|${{ env.LATEST_PRERELEASE_SECTION_REGEX }}|${{ env.LATEST_PRERELEASE_SECTION }}|g' ${{ env.PR_BODY_FILE_LOCATION }}
+          else  # a new section is added
+            echo -e '\n' >> ${{ env.PR_BODY_FILE_LOCATION }}
+            echo '---' >> ${{ env.PR_BODY_FILE_LOCATION }}
+            echo '${{ env.LATEST_PRERELEASE_SECTION }}' >> ${{ env.PR_BODY_FILE_LOCATION }}
```

Comment:
For some reason I couldn't get the `EOF` to work properly...see https://github.com/codegat-test-org/test/actions/runs/11947542405/job/33303660670#step:2:8

Reply:
Hmm maybe it expects the end-of-file (`EOL`) to be right at the beginning of the line. 
Have you tried with the ["indented" heredoc](https://unix.stackexchange.com/a/76483)?
Otherwise it's ok with multiple echo :D

Reply:
The "must use tabs" bit is a bit of a dealbreaker - vscode converts tabs to spaces and I don't want a section that only uses tabs :D